src/main/java/com/blogify/blogapi/service/UserService.java

```diff
@@ -14,4 +14,8 @@ public class UserService {
   public List<User> findAll() {
     return userRepository.findAll();
   }
+
+  public User save(User toSave) {
```

Comment:
Mark as `@Transactionnal` to prevent rollback when creating

Reply:
okay